Source/WebKit/Platform/IPC/StreamClientConnection.h

```diff
@@ -157,7 +157,7 @@ class StreamClientConnection final {
         Semaphore clientWait;
     };
     std::optional<Semaphores> m_semaphores;
-    unsigned m_maxBatchSize { 40 }; // Number of messages marked as StreamBatched to accumulate before notifying the server.
+    unsigned m_maxBatchSize { 20 }; // Number of messages marked as StreamBatched to accumulate before notifying the server.
     unsigned m_batchSize { 0 };
```

Comment:
Was this intended as part of the change? The ChangeLog didn't mention it and it seems a bit unrelated.

Reply:
It is! You're right that the ChangeLog definitely should mention it though, I'll do that now.